docs/source/notes/multiprocessing.rst

```diff
@@ -9,9 +9,12 @@ memory and will only send a handle to another process.
 
 .. note::
 
-    When a :class:`~torch.Tensor` is sent to another process, both
-    the :attr:`~torch.Tensor` data and :attr:`torch.Tensor.grad` are going to be
-    shared.
+    When a :class:`~torch.Tensor` is sent to another process, the
+    :attr:`~torch.Tensor` data is shared. If :attr:`torch.Tensor.grad` is
+    not ``None``, it is also shared. If :attr:`torch.Tensor.grad` is ``None``,
+    it is not shared and each shared copy of the :class:`~torch.Tensor`
```

Comment:
> "If :attr:`torch.Tensor.grad` is ``None``,
+    it is not shared and each shared copy of the :class:`~torch.Tensor`
+    accumulates separate gradients while training in their respective
+    processes."

should be rewritten. 

Once the Tensor is sent to the other process, if the `.grad` is initialized, it creates a standard process-specific Tensor and is not automatically shared across all process like how `.data` has been shared.